.azure/pipelines/ci.yml

```diff
@@ -149,12 +149,13 @@ stages:
       jobDisplayName: "Build: Windows x64/x86"
       agentOs: Windows
       steps:
-      - script: "echo ##vso[build.addbuildtag]daily-build"
-        condition: and(notin(variables['Build.Reason'], 'PullRequest'), notin(variables['DotNetFinalVersionKind'], 'release', 'prerelease'))
-        displayName: 'Set CI tags'
-      - script: "echo ##vso[build.addbuildtag]release-candidate"
-        condition: and(notin(variables['Build.Reason'], 'PullRequest'), in(variables['DotNetFinalVersionKind'], 'release', 'prerelease'))
-        displayName: 'Set CI tags'
+      - ${{ if notIn(variables['Build.Reason'], 'PullRequest') }}:
+        - script: "echo ##vso[build.addbuildtag]daily-build"
+          condition: notin(variables['DotNetFinalVersionKind'], 'release', 'prerelease')
+          displayName: 'Set CI tags'
+        - script: "echo ##vso[build.addbuildtag]release-candidate"
+          condition: in(variables['DotNetFinalVersionKind'], 'release', 'prerelease')
```

Comment:
@mmitche I can't find a build tagged with `release-candidate` in aspnetcore-ci-official and we set `$(DotNetFinalVersionKind)` in our `msbuild` code. Did we stop using this pipeline variable at some point❔ If yes, does it even matter for release/2.1❔

Reply:
I don't think this even existed in the 2.1 timeframe. For 3.1+ we set this in eng/Versions.props

Reply:
Alright, I removed the `condition`s using that variable and simplified this to just handle the `daily-build` tag, leaving it easy to find rolling (and manual) builds of all branches.